### DISCUSSION TITLE
webui doesn't see models pulled before in ollama CLI (both started from Docker Windows side; all latest)

### DISCUSSION THREAD

**Discussion by ai-bits** [open-webui/open-webui#770] [General]
# Bug Report

## Description

**Bug Summary:**
webui doesn't see models pulled before in ollama CLI (both started from Docker Windows side; all latest)

**Steps to Reproduce:**
ollama pull \<model\> # on ollama Windows cmd line
install / run webui on cmd line / browser. can't see \<model\>. no way to sync.
suspected different paths, but seems /root/.ollama/model in any case
d/l \<model\> from gui seems to overwrite already downloaded and has the exact same ID (GUID)
model path seems to be the same if I run ollama from the Docker Windows GUI / CLI side or use ollama on Ubuntu WSL (installed from sh) and start the gui in bash.

**Expected Behavior:**
ollama pull and gui d/l be in sync

## Environment

- **Operating System:** all latest Windows 11, Docker Desktop, WSL Ubuntu 22.04, ollama
- **Browser:** latest Chrome

**Comment by tjbck** [comment]
Hi, most likely the issue stems from having two Ollama instances running on the same machine. Could tell us about the installation method you used? The output for `docker ps` command would definitely help here for use to diagnose the issue.

**Comment by ai-bits** [comment]
Thanks for the lightspeed answer!
> most likely the issue stems from having two Ollama instances running on the same machine

Seems very valid suspicion. I installed ollama from .sh on Ubuntu 22.04 on WSL and then was intimidated by `systemctl edit ollama.service` from [Setting environment variables on Linux](https://github.com/ollama/ollama/blob/main/docs/faq.md), which brings up `/etc/systemd/system/ollama.service.d/override.conf` where I had no idea what to do.
So I pulled ollama on the Docker Desktop GUI on the Windows side and ran it from there, not realizing that I started the ollama installation on WSL Ubuntu when I opened a bash in Windows terminal.

I think calling the situation 'a bit complicated' is an understatement.
E.g. I just started Docker from the GUI on the Windows side and when I entered `docker ps` in Ubuntu bash I realized an ollama-webui container had been started.

I see the ollama and webui **images** in the Docker Desktop Windows GUI and I deleted the ollama container there after the experimentation yesterday.
**The bad pitfall** is that **the webui CONTAINER (running or not, started from the Windows or Ubuntu cmd line) is NOT VISIBLE there!**

Guess sample in case "what can go wrong does go wrong"!?
Remaining question: Why is the webui container not visible in the Docker Desktop Windows GUI, no matter where started from?

**Edit: Just saw that I CAN see the ollama-webui container in the Docker GUI on the MAC, where I installed ollama as app!**

Thanks
G.

**Comment by tjbck** [comment]
Could you also tell us the command you used to install the webui? If you're still facing this issue, I'd recommend you uninstall  both ollama and the webui and reinstall them.

**Reply by ai-bits** [reply]
It was always straight from the docs:
`docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v ollama-webui:/app/backend/data --name ollama-webui --restart always ghcr.io/ollama-webui/ollama-webui:main`

But that's moot now as I just found the culprit, my bad:
I had somehow fiddled an ID into the Search on the Containers GUI page when I checked for details of the Ollama container. As I had deleted all other containers but the Ollama-related two the search string went undetected.

Sorry for the false alarm.
Thanks
G.
